js/calendar.js

```diff
@@ -877,6 +878,10 @@ if(!String.prototype.formatNum) {
 	Calendar.prototype._loadEvents = function() {
 		var self = this;
 		var source = null;
+		var auth_header = null;
+		if('auth_header' in this.options && this.options.auth_header !== '') {
+			auth_header = this.options.auth_header;
+		}
```

Comment:
I do not think you need this 881-884 block of code. You have already set it in defaults.